config/settings.yml

```diff
@@ -29,3 +29,5 @@ plugins:
     default: ''
     type: list
     list_type: compact
+  openid_connect_claims:
+    default: ''
```

Comment:
Please can you add a trailing newline here

Reply:
Yep, done!